fix!: change all_* and any_* to take *apps instead of a list (#116)
canonical/jubilant: docs/tutorial/getting-started.md

```python
def test_deploy(juju: jubilant.Juju):
    juju.deploy('snappass-test')
    juju.wait(jubilant.all_active)

>     # Or wait more specifically for only 'snappass-test' to be active:

Reviewer: Similar to my other comment, how about:

```suggestion
    # Or specifically wait for 'snappass-test' to be active (ignoring other applications):
```

> For example, to wait till `myapp` is active and `yourapp` is blocked, and to raise an error if any app or unit goes into error state:

Reviewer: Commenting on the code below:

```
        lambda status: (
            status.apps['myapp'].is_active and
            status.apps['yourapp'].is_blocked
        ),
```

Is there a reason you left this as-is instead of switching to `jubilant.all_active(status, 'myapp') and jubilant.all_blocked(status, 'yourapp')` like you did in other places?

Author: This is in the context of introducing the `is_*` properties (see the paragraphs above this). However, I've added a sentence to the paragraph introducing them to clarify the distinction between `is_*` and `all_*`/`any_*`.

Reviewer: That makes sense, thanks. I think the [Use a custom wait condition](https://canonical-jubilant--116.com.readthedocs.build/tutorial/getting-started/#use-a-custom-wait-condition) looks great.
